src/main/java/com/fasterxml/jackson/core/util/BufferRecyclerPool.java

```diff
@@ -285,4 +287,70 @@ static class Node {
             }
         }
     }
+
+    /**
+     * {@link BufferRecyclerPool} implementation that uses
+     * a bounded queue for recycling instances.
+     */
+    class BoundedPool implements BufferRecyclerPool
+    {
+        private static final long serialVersionUID = 1L;
+
+        private static final BufferRecyclerPool SHARED = new BoundedPool(16);
```

Comment:
16? If this is number of instances to share globally, should probably be higher, maybe 100?

Either way, should probably be defined as constant, and mentioned in Javadocs.

Reply:
Changed to 100 for now.